x/feeabstraction/module.go

```diff
@@ -134,6 +134,13 @@ func (AppModule) QuerierRoute() string { return types.QuerierRoute }
 func (am AppModule) RegisterServices(c module.Configurator) {
 	types.RegisterMsgServer(c.MsgServer(), keeper.NewMsgServer(am.keeper))
 	types.RegisterQueryServer(c.QueryServer(), keeper.NewQuerier(am.keeper))
```

Comment:
Don't forget to update the consensus version from the top. We should go from 1 to 2